### ISSUE TITLE
Issues with FSE

### ISSUE THREAD

**Issue by strarsis** [joanrodas/sage10-bbpress#1]
[`get_page_template()`](https://github.com/joanrodas/sage10-bbpress/blob/main/src/bbpress-compat.php#L7) returns the canvas template instead of a normal template in Gutenberg FSE themes.

**Comment by joanrodas** [comment]
You are right. I should probably use locate_block_template() instead. I'll check it.
